### PR TITLE
Create abload.de.xml

### PR DESCRIPTION
**Pull request by heuser443**

### CODE REVIEW

**Comment by ihave100openPRs** [comment]
Please document its subdomains that have invalid certificates such as h13.abload.de

**Comment by heuser443** [comment]
I do not see any use of mentioning a probably incomplete list of internal subdomains, of which some have likely just accidentially found a way into search engine indexes without actually being intended/used for serving web pages, but there you go.

**Comment by heuser443** [comment]
Hm, after failing without an obvious reason on my previous commit, now it looks like my last commit did not trigger a Travis CI build at all. I think something is wrong with the build/checking process.

**Comment by ihave100openPRs** [comment]
@heuser443 You can close and then re-open this PR so that Travis gets triggered.

**Comment by heuser443** [comment]
@ihave100openPRs, thanks for the advice. Check was successful now, as expected. Must have been a temporary issue with Travis CI in my two attempts before.

File: src/chrome/content/rules/abload.de.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	Invalid certificate:
+		h1.abload.de
+		h2.abload.de
+		...
+		h19.abload.de
+		pool.abload.de
+
+-->
+
+<ruleset name="abload.de">
```
Comment (J0WI) [inline src/chrome/content/rules/abload.de.xml]: Please rename the rule to `abload.de (partial)`.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**